js/_website/src/lib/templates/gradio/05_modals/error.svx

```diff
@@ -17,13 +17,19 @@
 
 <!--- Usage -->
 ```python
-gradio.Error(···)
+raise gradio.Error("An error occured 💥!", duration=5)
 ```
 
 <!--- Description -->
 ### Description
 ## {@html style_formatted_text(obj.description)}
 
+## You can control for how long the error message is displayed with the `duration` parameter. If it's `None`, the message will be displayed forever until the user closes it. If it's a number, it will be shown for that many seconds. 
+
+## Below is a demo of how different values of duration control the error, info, and warning messages. You can see the code [here](https://huggingface.co/spaces/freddyaboulton/gradio-error-duration/blob/244331cf53f6b5fa2fd406ece3bf55c6ccb9f5f2/app.py#L17).
```

Comment:
@aliabd The link here is not being formatted as a link. How can I fix it?

<img width="859" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/82d76159-cbbb-4685-81c9-f046120d2b9f">

Reply:
pushed a commit that fixed it. also styling clash. 

what's happening with these issues is that i had overrided some default styling rules to design stuff (for example the side bar is a ul but we don't want bullet points there) and now we need them for the content.

Reply:
Thank you!